homeassistant/components/camera/nest.py

```diff
@@ -76,7 +77,34 @@ def brand(self):
         """Return the brand of the camera."""
         return NEST_BRAND
 
-    # This doesn't seem to be getting called regularly, for some reason
+    @property
+    def supported_features(self):
+        """Nest Cam support turn on and off."""
+        return SUPPORT_ON_OFF
+
+    @property
+    def is_on(self):
+        """Return true if on."""
+        return self._online and self._is_streaming
+
+    def turn_off(self):
+        """Turn off camera."""
+        _LOGGER.debug('Turn off camera %s', self._name)
+        # Calling Nest API in is_streaming setter
+        self.device.is_streaming = False
+        self.schedule_update_ha_state(True)
```

Comment:
Since this is a polling entity, this line isn't needed.

Reply:
You are right. Just test out on real device, after this statement, camera still need few seconds to finish process and return is_streaming = True. The update statement is useless here anyway.